ova/generate_ova.sh

```diff
@@ -34,12 +36,11 @@ help () {
 
     echo
     echo "Usage: $0 [OPTIONS]"
-    echo
-    echo "  -b, --build            [Required] Build the OVA and OVF."
     echo "  -v, --version          [Required] Version of wazuh to install on VM."
-    echo "  -e, --elastic-version  [Required] Elastic version to download inside VM."
-    echo "  -r, --repository       [Required] Status of the packages [stable/unstable]"
-    echo "  -d, --directory        [Optional] Where will be installed manager. Default /var/ossec"
+    echo "  -o, --opendistro       [Required] Version of Open Distro for Elasticsearch."
+    echo "  -f, --filebeat         [Required] Filebeat version to download inside VM."
```

Comment:
```suggestion
    echo "  -f, --filebeat         [Required] Filebeat's version."
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a